be/src/io/cache/block_file_cache.cpp

```diff
@@ -1302,8 +1347,18 @@ void BlockFileCache::remove(FileBlockSPtr file_block, T& cache_lock, U& block_lo
     *_num_removed_blocks << 1;
 }
 
+void BlockFileCache::recycle_stale_rowset_async_bottom_half() {
+    FileCacheKey key;
+    while (_recycle_keys->pop(key)) {
+        Status st = _storage->remove(key);
+        if (!st.ok()) {
+            LOG_WARNING("").error(st);
```

Comment:
put some key word in the log e.g. 
"failed to remove when recycle_stale_rowset_async_bottom_half, key = {}"  ...